test/samples/json/apip-data.json

```diff
@@ -0,0 +1,894 @@
+              "base": 10,
+              "expectedLength": 240,
+              "minStrings": 0,
+              "format": "plain",
+              "class": "height-tall behavior-highlight"
+            },
+            "debug": {
+              "relatedItem": "item_5df0be6b661e2141526766"
+            },
+            "choices": [],
+            "prompt": {
+              "serial": "container_containerstatic_5df0be6b67201000810208",
+              "body": "",
+              "elements": [],
+              "debug": {
+                "relatedItem": "item_5df0be6b661e2141526766"
+              }
+            }
+          }
+        },
+        "debug": {
+          "relatedItem": "item_5df0be6b661e2141526766"
+        }
+      },
+      "debug": {
+        "relatedItem": "item_5df0be6b661e2141526766"
+      },
+      "namespaces": {
+        "": "http:\/\/www.imsglobal.org\/xsd\/imsqti_v2p1",
+        "xi": "http:\/\/www.w3.org\/2001\/XInclude",
+        "xsi": "http:\/\/www.w3.org\/2001\/XMLSchema-instance"
+      },
+      "schemaLocations": {
+        "http:\/\/www.imsglobal.org\/xsd\/imsqti_v2p1": "http:\/\/www.imsglobal.org\/xsd\/qti\/qtiv2p1\/imsqti_v2p1p2.xsd",
+        "http:\/\/www.w3.org\/2001\/XInclude": "http:\/\/www.imsglobal.org\/xsd\/w3\/2001\/XInclude.xsd",
+        "http:\/\/www.w3.org\/1998\/Math\/MathML": "http:\/\/www.w3.org\/Math\/XMLSchema\/mathml2\/mathml2.xsd",
+        "http:\/\/www.imsglobal.org\/xsd\/apip\/apipv1p0\/qtiitem\/imsqti_v2p1": "http:\/\/www.imsglobal.org\/profile\/apip\/apipv1p0\/apipv1p0_qtiitemv2p1_v1p0.xsd"
+      },
+      "stylesheets": [],
+      "outcomes": {
+        "outcomedeclaration_5df0be6b66614536126421": {
+          "identifier": "SCORE",
+          "serial": "outcomedeclaration_5df0be6b66614536126421",
+          "qtiClass": "outcomeDeclaration",
+          "attributes": {
+            "identifier": "SCORE",
+            "cardinality": "single",
+            "baseType": "float",
+            "interpretation": "scoreSystem:human responseIdentifier:RESP",
+            "normalMaximum": 0
+          },
+          "debug": {
+            "relatedItem": "item_5df0be6b661e2141526766"
+          },
+          "defaultValue": "0"
+        }
+      },
+      "responses": {
+        "responsedeclaration_5df0be6b664d3594302331": {
+          "identifier": "RESP",
+          "serial": "responsedeclaration_5df0be6b664d3594302331",
+          "qtiClass": "responseDeclaration",
+          "attributes": {
+            "identifier": "RESP",
+            "cardinality": "single",
+            "baseType": "string"
+          },
+          "debug": {
+            "relatedItem": "item_5df0be6b661e2141526766"
+          },
+          "defaultValue": []
+        }
+      },
+      "feedbacks": [],
+      "apipAccessibility": {
+        "companionMaterialsInfo": [],
+        "inclusionOrder": {
+          "textGraphicsDefaultOrder": {
+            "elementOrder": [
+              {
+                "@attributes": {
```

Comment:
Maybe nothing directly related to your task, but what are those weird and useless properties `@attributes`?
It should be rather:
```json
...
{
    "identifierRef": "ae001",
    "order": "1"
},
{
    "identifierRef": "ae002",
    "order": "2"
},
...
```